### PR TITLE
Cron job at 01:00 not every minute of the first hour

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *`* 1 * * *` means "At every minute past hour 1."*

## Solution
- *Trigger the job at 01am*

You can play with this [website](https://crontab.guru/) for a deeper understanding of cron

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
*If you have access to [App Continuous Integration for packagers](https://yunohost.org/#/packaging_apps_ci) you can provide a link to the package_check results like below, replacing '-NUM-' in this link by the PR number and USERNAME by your username on the ci-apps-dev. Or you provide a screenshot or a pastebin of the results*

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/aeneria_ynh%20PR-NUM-%20(USERNAME)/)

### CODE REVIEW

**Comment by SimonMellerin** [comment]
ouch' this is a huge forgot :/

This cron should be launch something like every 20minutes, so it should be `*/20 * * * *`

Thanks for the report and the PR !

File: conf/aeneria.cron [hunk 1]
```diff
@@ -1 +1 @@
-* 1 * * * #USER# php7.3 #DESTDIR#/bin/console aeneria:fetch-data
+0 1 * * * #USER# php7.3 #DESTDIR#/bin/console aeneria:fetch-data
```
Comment (kay0u) [inline conf/aeneria.cron]:
```suggestion
*/20 * * * * #USER# php7.3 #DESTDIR#/bin/console aeneria:fetch-data
```

**Comment by kay0u** [comment]
Fixed :-)

Thanks for this awesome app (and YNH package)

**Comment by kay0u** [comment]
(FYI, the targeted branch was testing, not master ;-))

**Pull request merged**